ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -408,114 +413,130 @@ export default class ConfirmTransactionBase extends Component {
+                      </InfoTooltip>
+                    </>
                   ) : (
-                    ''
-                  )}
-                  {supportsEIP1559 && (
-                    <GasTiming
-                      maxPriorityFeePerGas={hexWEIToDecGWEI(
-                        maxPriorityFeePerGas ||
-                          txData.txParams.maxPriorityFeePerGas,
-                      )}
-                      maxFeePerGas={hexWEIToDecGWEI(
-                        maxFeePerGas || txData.txParams.maxFeePerGas,
-                      )}
+                    <>
+                      {t('transactionDetailGasHeading')}
+                      <InfoTooltip
+                        contentText={
+                          <>
+                            <p>
+                              {t('transactionDetailGasTooltipIntro', [
+                                isMainnet ? t('networkNameEthereum') : '',
+                              ])}
+                            </p>
+                            <p>{t('transactionDetailGasTooltipExplanation')}</p>
+                            <p>
+                              <a
+                                href="https://community.metamask.io/t/what-is-gas-why-do-transactions-take-so-long/3172"
+                                target="_blank"
+                                rel="noopener noreferrer"
+                              >
+                                {t('transactionDetailGasTooltipConversion')}
+                              </a>
+                            </p>
+                          </>
+                        }
+                        position="top"
+                      >
+                        <i className="fa fa-info-circle" />
+                      </InfoTooltip>
+                    </>
+                  )
+                }
+                detailTitleColor={COLORS.BLACK}
+                detailText={
+                  <div className="confirm-page-container-content__currency-container">
+                    {renderHeartBeatIfNotInTest()}
+                    <UserPreferencedCurrencyDisplay
+                      type={SECONDARY}
+                      value={hexMinimumTransactionFee}
+                      hideLabel={Boolean(useNativeCurrencyAsPrimaryCurrency)}
                     />
-                  )}
-                </>
-              }
-            />,
+                  </div>
+                }
+                detailTotal={
+                  <div className="confirm-page-container-content__currency-container">
+                    {renderHeartBeatIfNotInTest()}
+                    <UserPreferencedCurrencyDisplay
+                      type={PRIMARY}
+                      value={hexMinimumTransactionFee}
+                      hideLabel={!useNativeCurrencyAsPrimaryCurrency}
+                    />
+                  </div>
+                }
+                subText={t('editGasSubTextFee', [
+                  <b key="editGasSubTextFeeLabel">
```

Comment:
Should we use `<Typography fontWeight={TYPOGRAPHY.BOLD}>` here?

Reply:
Same as above here, in addition difference if that Typography creates block where <b> is inline, typography component also supports it: https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/ui/typography/typography.scss#L7